.clang-format

```diff
@@ -5,35 +5,106 @@
 
 BasedOnStyle: "LLVM"
 
-AlignOperands:   true
+AccessModifierOffset: -2
+
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignEscapedNewlines: Left
+AlignOperands: true
 AlignTrailingComments: true
-AllowAllParametersOfDeclarationOnNextLine: true
+
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+
+AlwaysBreakAfterReturnType: All
+AlwaysBreakBeforeMultilineStrings: false
 AlwaysBreakTemplateDeclarations: true
 
 BinPackArguments: false
 BinPackParameters: false
 
-BreakBeforeBraces: Custom
 BraceWrapping:
   AfterClass: true
   AfterControlStatement: true
   AfterEnum: true
+  AfterExternBlock: true
   AfterFunction: true
   AfterNamespace: true
-  AfterObjCDeclaration: false
   AfterStruct: true
   AfterUnion: true
   BeforeCatch: true
   BeforeElse: true
   IndentBraces: true
+  SplitEmptyFunction: false
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+
+BreakBeforeBinaryOperators: All
+BreakBeforeBraces: Custom
+BreakBeforeInheritanceComma: false
+BreakBeforeTernaryOperators: false
+BreakConstructorInitializers: BeforeColon
+BreakStringLiterals: false
+
+ColumnLimit: 80
+
+CompactNamespaces: false
 
-ColumnLimit:     80
 ConstructorInitializerAllOnOneLineOrOnePerLine: true
 ConstructorInitializerIndentWidth: 2
+
 ContinuationIndentWidth: 2
-IndentWidth:     2
+
+Cpp11BracedListStyle: true
+
+DerivePointerAlignment: false
+
+DisableFormat: false
+
+FixNamespaceComments: true
+
+IncludeBlocks: Preserve
+
+IndentCaseLabels: true
+IndentPPDirectives: AfterHash
+IndentWidth: 2
+
+IndentWrappedFunctionNames: false
+
+KeepEmptyLinesAtTheStartOfBlocks: false
+
+Language: Cpp
+
+MaxEmptyLinesToKeep: 1
+
 NamespaceIndentation: All
-Standard:        Cpp11
 
-TabWidth:        4
-UseTab:          Never
+PointerAlignment: Left
```

Comment:
For the other two options see https://github.com/masterleinad/dealii/pull/10 and https://github.com/masterleinad/dealii/pull/9

Reply:
by not working you mean cases like

```
   RightHandSide<dim>::vector_value_list(
 -    const std::vector<Point<dim>>& points,
 -    std::vector<Vector<double>>&   value_list) const
 +    const std::vector<Point<dim>> &points,
 +    std::vector<Vector<double>> &  value_list) const
```

? 

Personally, i would still prefer `right` to `left`.

Reply:
Also prefer right